jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
I am curious what exact performance benefit this gives for a stateless functional component, and am always hesitant to take on any React-based library that might inhibit us from upgrading React later.